src/components/ReportWelcomeText.tsx

```diff
@@ -1,92 +1,65 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
You should be able to remove `lodash` completely in this file:
```ts
{!_.isEmpty(pronouns) && <Text>{` (${pronouns})`}</Text>}
```
```ts
{!!pronouns && <Text>{` (${pronouns})`}</Text>}
```

Reply:
Thanks, addressed by [3fad7ce](https://github.com/Expensify/App/pull/33251/commits/3fad7ceb9c9cf49242e2e42556ca5f36c296d56b)